### PR TITLE
My Portfolio

### PR DESCRIPTION
**Pull request by SarahA2020**
Includes media queries and is working in Firefox

### CODE REVIEW

File: myPortfolio.html [hunk 1]
```diff
@@ -41,6 +44,15 @@ <h2 class="sectionHeading">Portfolio</h2>
 		</div>	
 	</section>
 
+	<footer id="jumpToContact">
+		<i class="material-icons" id="btnCall">call</i>
+		<i class="material-icons" id="btnEmail">email</i>
+		<i class="material-icons" id="btnFacebook">facebook</i>
```
Comment (davin2020) [inline myPortfolio.html]: Dont forget to add your links in here

File: myPortfolio.html [hunk 2]
```diff
@@ -6,32 +6,35 @@
...
-	<section class="sectionStyling">
+	<section class="sectionStyling" id="jumpToAboutMe">
 		<div class="textContainer">
 			<div class="centeredContent">
 				<h2 class="sectionHeading">About Me</h2>
-				<p class="sectionText">In hac habitasse platea dictumst. Aliquam tincidunt massa vel ultricies congue. Praesent fringilla sodales est nec iaculis. Mauris eu ultrices elit. Phasellus non massa gravida, blandit magna eu, fermentum dui. Sed gravida elementum faucibus. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.<p>
+				<p class="sectionText">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce egestas imperdiet accumsan. Vestibulum et lacus ut justo dapibus bibendum lobortis eu diam. Donec arcu velit, vehicula aliquet nulla nec, auctor consequat lacus.<p>
 			</div>
 		</div>
 		<img class="aboutMeImage" src="https://cdn.pixabay.com/photo/2017/11/27/21/31/computer-2982270_960_720.jpg">
```
Comment (davin2020) [inline myPortfolio.html:34]: Image of laptop looks squashed width-wise in Safari when you shrink the browser.
Screenshot 2020-09-09 at 14 56 23

File: myPortfolio.html [hunk 3]
```diff
@@ -6,32 +6,35 @@
...
+	
 </head>
 <body>
 	<nav class="navigation">
-		<a href="#">About Me</a>
-		<a href="#">Portfolio</a>
-		<a href="#">Contact</a>
+		<a href="#jumpToAboutMe">About Me</a>
+		<a href="#jumpToPortfolio">Portfolio</a>
+		<a href="#jumpToContact">Contact</a>
```
Comment (davin2020) [inline myPortfolio.html:18]: You might like to consider a media query for the height of the nav when the screen is narrow
Screenshot 2020-09-09 at 15 11 43

**Review by davin2020 — CHANGES_REQUESTED** [review]
A few minor issues with images but otherwise ok

File: myPortfolio.html (hunk 2 above)
Comment (SarahA2020) [reply myPortfolio.html:34]: Thanks, I haven't got to cross-browser testing yet. Will look at this tomorrow.

File: myPortfolio.html [hunk 4]
```diff
@@ -3,35 +3,40 @@
 <head>
 	<meta charset="utf-8">
 	<title>Sarah Allen Portfolio</title>
+	<script src="https://kit.fontawesome.com/80b0dca2ff.js"
+		 crossorigin="anonymous"></script>
```
Comment (jennymegan) [inline myPortfolio.html]: unnecessary line break

File: myPortfolio.html [hunk 5]
```diff
@@ -3,35 +3,40 @@
...
 	<link href="normalise.css" type="text/css" rel="stylesheet">
 	<link href="myPortfolio.css" type="text/css" rel="stylesheet">
 	<link href="https://fonts.googleapis.com/css2?family=Kufam:wght@400;600&display=swap" rel="stylesheet">
-	<meta name=’viewport” content=”width=device-width, initial-scale=0.1”>
+	<link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+	      rel="stylesheet">
```
Comment (jennymegan) [inline myPortfolio.html]: another unnecessary line break

File: myPortfolio.html [hunk 6]
```diff
@@ -3,35 +3,40 @@
...
 		<p class="pseudoText">SA</p>		
 	</header>
 	
-	<section class="sectionStyling">
+	<section class="sectionStyling" id="jumpToAboutMe">
 		<div class="textContainer">
 			<div class="centeredContent">
 				<h2 class="sectionHeading">About Me</h2>
-				<p class="sectionText">In hac habitasse platea dictumst. Aliquam tincidunt massa vel ultricies congue. Praesent fringilla sodales est nec iaculis. Mauris eu ultrices elit. Phasellus non massa gravida, blandit magna eu, fermentum dui. Sed gravida elementum faucibus. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.<p>
+				<p class="sectionText">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce egestas imperdiet accumsan. Vestibulum et lacus ut justo dapibus bibendum lobortis eu diam. Donec arcu velit, vehicula aliquet nulla nec, auctor consequat lacus.<p>
```
Comment (jennymegan) [inline myPortfolio.html]: The p tag isn't closing (needs a fwd slash)

File: myPortfolio.html [hunk 7]
```diff
@@ -41,6 +46,25 @@ <h2 class="sectionHeading">Portfolio</h2>
...
+			<i class="fas fa-mobile-alt"></i>
+		</a>
+		<a href="mailto:sairs_77@hotmail.com">
+			<i class="fas fa-envelope"></i>
+		</a>
+		<a href="https://github.com/Sass77" class="noPadding" target="_blank">
+			<i class="fab fa-github"></i>
+		</a>
+			
+
```
Comment (jennymegan) [inline myPortfolio.html]: unnecessary space here

File: myPortfolio.css [hunk 8]
```diff
@@ -24,13 +31,43 @@ nav a {
...
+}
+
 nav a:hover {
 	color: #e1ac28;
 }
 
 header {
 	background-image: url("https://cdn.pixabay.com/photo/2020/04/27/17/58/petals-5100957_960_720.jpg");
 	background-size: cover;
+	background-position: center;
```
Comment (jennymegan) [inline myPortfolio.css:67]: on line 76 there is another line break that can go

File: myPortfolio.css [hunk 9]
```diff
@@ -105,20 +183,118 @@ p.overlap {
...
 }
 
+@media screen and (max-width: 1000px) {
+	.aboutMeImage {
+		width: 100%;
+	}
+}
 
 .portfolioImage {
+
```
Comment (jennymegan) [inline myPortfolio.css]: line break not needed

File: myPortfolio.css [hunk 10]
```diff
@@ -105,20 +183,118 @@ p.overlap {
...
+	color: #54c9c4;
+}
+
+.fab {
+	font-size: 3rem;
+}
+
+.fas {
+	font-size: 3rem;
+
```
Comment (jennymegan) [inline myPortfolio.css]: line break not needed

**Review by jennymegan — APPROVED** [review]

**Review by jennymegan — APPROVED** [review]

**Review by jennymegan — APPROVED** [review]

**Pull request merged**